resources/definitions/twotrees_sapphirepro.def.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": 2,
+    "name": "TwoTrees Sapphire Pro",
+    "inherits": "fdmprinter",
+    "metadata":
+    {
+        "visible": true,
+        "author": "MLoewe",
+        "manufacturer": "TwoTrees",
+        "file_formats": "text/x-gcode",
+        "platform": "twotrees_sapphirepro.stl",
+        "platform_texture": "twotrees_sapphirepro.png",
+        "platform_offset": [0, 0, 0],
+        "machine_extruder_trains":
+		{
+			"0": "twotrees_sapphirepro_extruder_0",
+			"1": "twotrees_sapphirepro_extruder_1"
+		}
+    },
+    "overrides":
+    {
+        "machine_name": { "default_value": "Two Trees Sapphire Pro" },
+        "machine_heated_bed": { "default_value": true },
+        "machine_width": { "default_value": 220 },
+        "machine_depth": { "default_value": 220 },
+        "machine_height": { "default_value": 220 },
```

Comment:
The [machine's official site](https://www.twotrees3dprinter.com/#/app/productxq/2) mentions both L235xW235xH235mm and L220xW220xH220mm as machine dimentions. Can you verify that the L220xW220xH220mm is the correct one?